pytensor/graph/basic.py

```diff
@@ -970,6 +970,132 @@ def applys_between(
     )
 
 
+def truncated_graph_inputs(
+    outputs: Sequence[Variable],
+    ancestors_to_include: Optional[Collection[Variable]] = None,
+) -> List[Variable]:
+    """Get the conditional subset for outputs.
+
+
+    Conditional subset is a minimal set of variables required to compute outputs.
```

Comment:
This description still isn't clear for me. It doesn't mention what happens with `ancestors_to_include` which is the key thing of this method

Reply:
updated